### PR TITLE
Fix postgresql version and ssh idempotency

### PR DESCRIPTION
**Pull request by lrsksr**
- **fix(postgresql): look at PG_VERSION file instead of the container**
- **chore(postgresql): add check for empty directory in initialization**

### CODE REVIEW

File: roles/postgresql/tasks/main.yml [hunk 1]
```diff
@@ -58,29 +46,9 @@
   register: "postgresql_image_pulled"
   tags: ["prepare", "prepare-postgresql"]
 
-- name: "Set container version label to version of pulled image"
```
Comment (sebastian-de) [inline roles/postgresql/tasks/main.yml:61]: The version label defaults to `postgresql_container_version`, so without this, only the major version will be in the label. I'm therefore in favor of keeping this.

**Review by sebastian-de — COMMENTED** [review]
apart from that LGTM.

**Review by sebastian-de — APPROVED** [review]

File: roles/ssh/tasks/main.yml [hunk 2]
```diff
@@ -221,7 +221,7 @@
   ansible.builtin.lineinfile:
     <<: *sshd_config_args
     regexp: "^#?PubkeyAcceptedKeyTypes"
-    line: "PubkeyAcceptedKeyTypes {{ ssh_preset.allowed_pubkey_algos | intersect(queried_pubkey_algos.stdout_lines) | join(',') }}"
+    line: "PubkeyAcceptedKeyTypes {{ ssh_preset.allowed_pubkey_algos | intersect(queried_pubkey_algos.stdout_lines) | sort | join(',') }}"
```
Comment (jcgruenhage) [inline roles/ssh/tasks/main.yml:224]: *Ideally*, we would keep the order as they are in the preset. Right now, this prefers sha256 over sha512 due to alphabetic sorting.

Comment (lrsksr) [reply roles/ssh/tasks/main.yml]: Added `reverse=true` to the sort until this is fixed, as the stronger algorithms are *generally* later in the alphabet. If an algorithm is deemed not secure enough, it should be removed from the list.

**Pull request merged**